packages/bind-mount/package.json

```diff
@@ -5,7 +5,8 @@
   "types": "index.d.ts",
   "dependencies": {
     "bindings": "^1.5.0",
-    "nan": "^2.17.0"
+    "nan": "^2.17.0",
+    "node-gyp": "^9.3.1"
```

Comment:
This was previously an implicit dependency, but Yarn now requires it to be explicit. This is a best practice anyways.